test/autograd/test_train.py

```diff
@@ -45,6 +45,6 @@ def test_binary_classification(self):
         y_pred = model(X).data
 
         # compare y_pred and y on the classification accuracy
-        accuracy = sum((y_pred > 0.5).astype(int) == y) / X.shape[0]
```

Comment:
y_pred is now (N, 1) due to all gradients are now initialized as vectors/matrices instead of scalars, so we need to squeeze it to compute accuracy